open_schools_platform/organization_management/employees/views.py

```diff
@@ -41,7 +43,7 @@ def get(self, request, *args, **kwargs):
         return response
 
 
-class EmployeeQueriesListApi(ApiAuthMixin, ListAPIView):
+class EmployeeQueriesListApi(ApiAuthMixin, APIView):
     pagination_class = ApiListPagination
     serializer_class = EmployeeProfileQuerySerializer
 
```

Comment:
more information in swagger, we should delete pagination and serializer class if we don't use ListApiView